awx/__init__.py

```diff
@@ -21,6 +21,30 @@
     MODE = 'production'
 
 
+import django
+import hashlib
+from django.utils.encoding import force_bytes
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.backends.base import schema
+
+class FipsBaseDatabaseSchemaEditor(BaseDatabaseSchemaEditor):
+    
+    @classmethod
+    def _digest(cls, *args):
+        """
+        Generates a 32-bit digest of a set of arguments that can be used to
+        shorten identifying names.
+        """
+        h = hashlib.md5(usedforsecurity=False)
+        for arg in args:
+            h.update(force_bytes(arg))
+        return h.hexdigest()[:8]
+        
+schema.BaseDatabaseSchemaEditor = FipsBaseDatabaseSchemaEditor
```

Comment:
I stared at DH's backend/schema implementation, and while I don't love the import-time monkey-patching, I can't see an easier/less gross way to do this.  _Maybe_ we could define a custom backend that extended postgres, but that looks like it's going to be more trouble than it's worth.

👍